triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -527,7 +527,8 @@ class TriggerServiceTestWithDb
         aliceTrigger = triggerIds.head
         // Currently the logs aren't persisted so we can check that the trigger was restarted by
         // inspecting the new log.
-        _ <- assertTriggerStatus(uri, aliceTrigger, _.last == "running", 30)
+        _ <- assertTriggerStatus(uri, aliceTrigger, _.nonEmpty)
+        _ <- assertTriggerStatus(uri, aliceTrigger, _.last == "running")
```

Comment:
I’m not sure I understand this change. `assertTriggerStatus` is an `eventually`. Shouldn’t `.last` fail until it is non-empty?

Reply:
Yeah, I had a vague hypothesis about what was going on which turned out to be wrong once I read how `eventually` works. So now I don't think this will do anything